warehouse/models/staging/gtfs/gtfs_quality/stg_gtfs_schedule__validation_notices.sql

```diff
@@ -0,0 +1,22 @@
+WITH raw_validation_notices AS (
+    SELECT * FROM {{ source('external_gtfs_schedule', 'validation_notices') }}
+),
+
+stg_gtfs_schedule__validation_notices AS (
```

Comment:
see overall comment re: unnesting the notices at some point... would that be an intermediate table on top of this...? or a separate table for sample notices? 🤔

Reply:
I think an intermediate table, because it's changing grain.